### ISSUE TITLE
Preprocessing tutorial: plot X-Y scale defaults inappropriate for Waters data?

### ISSUE THREAD

**Issue by Phylloxera** [sneumann/xcms#470]
![TIC](https://user-images.githubusercontent.com/13121075/80519535-c04d4d00-894d-11ea-889a-45b8dbbae9d0.png)
![all_chromatograms](https://user-images.githubusercontent.com/13121075/80519538-c2afa700-894d-11ea-91f9-b54203939fc9.png)

Hello,

I'm not really sure if my title describes the underlying issue... I'm looking over issues [430](https://github.com/sneumann/xcms/issues/430) and [451](https://github.com/sneumann/xcms/issues/451) and centwave parameter definitions [here](https://github.com/cbroeckl/csu.pmf.tools/blob/master/R/pmf-xcms_new.R) for downstream steps... but where I'm stuck for now is the results of:
`plot(bpis, col = group_colors[raw_data$sample_group])`
and
`
boxplot(tc, col = group_colors[raw_data$sample_group],
        ylab = "intensity", main = "Total ion current")
`
In the first case, the results of 
`head(rtime(bpis[1,1])) `
> F01.S0001 F01.S0002 F01.S0003 F01.S0004 F01.S0005 
>     1.617     1.918     2.132     2.346     2.561 
> F01.S0006 
>     2.775

make sense to me, while the plot X scale seems to instead match the mass range (second plot).
In the second case (first plot), the boxplots do not seem informative in the same way they are in the [tutorial](https://www.bioconductor.org/packages/devel/bioc/vignettes/xcms/inst/doc/xcms.html#8_additional_details_and_notes). I think I remember reading that waters intensity values are orders of magnitude higher than other data types, is that an issue here? If so, can someone steer me in the right direction regarding appropriate adjustments to the plot commands?

Thanks, Aaron

**Comment by jorainer** [comment]
For the `boxplot` I would suggest to plot the log2 transformed intensities instead of the natural scale values (something like `boxplot(lapply(tc, log2), ...`). For the base peak chromatogram, what puzzles me is the *spikes* that you have at regular intervals - are these the lock mass scans?

Regarding your concern over the mass range or the retention time, the BPC plot shows the retention time on the x-axis (in seconds). The range of the m/z is shown as the title of the plot. In your particular case it seems that m/z range and retention time range are similar.

**Comment by Phylloxera** [comment]
Yes, those are lock mass scans. Clearly, I have multiple issues :(
I've got my msconvert command for taking off the lockmass nailed down and I'll give the log2 transformation a try for the boxplot. Maybe I'll be able to visualize the first plot better if I can split it into multiple smaller RT windows. I'll think on that.
Thanks!
